src/generate-binaries.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
+set -ex
+# maven plugin version
+VER=$1
+# path to the golang source
+SRC=$2
+IMG_NAME="durable-task-binary-generator"
+docker build --build-arg PLATFORM="buster" -t ${IMG_NAME}:${VER} .
+docker run -i --rm --mount type=bind,src=${SRC},dst=/durabletask ${IMG_NAME}:${VER} /durabletask/test-and-compile.sh ${VER}
+docker rmi ${IMG_NAME}:${VER}
```

Comment:
IIUC you are trying to produce binaries from the `Dockerfile` into the workspace? If so, this idiom is too complex and fragile. Assuming Docker is not years old, just use `docker build --output`, with multi-stage builds if necessary.

Reply:
oh wow, i wasn't aware of `--output` before. THANK YOU

Reply:
switches to docker build --output in 48efa4fe70f4ba2b9a29944253e7934a4afd8406
At least for linux. no BuildKit support for windows containers yet.